testbed/addons/frill.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
+kind: Kustomization
```

Comment:
IIUC, you don't actually need to use Kustomize for a Kustomization CRD (i.e., you can skip the kustomize.yaml and just do raw YAML).